src/SmartComponents/Modals/UnassignSystemsModal.test.js

```diff
@@ -62,35 +54,25 @@ describe('UnassignSystemsModal', () => {
             })
         );
 
-        await wrapper.update().find(Modal).props().actions[0].props.onClick();
-        expect(addNotification).toHaveBeenCalledWith(
-            patchSetUnassignSystemsNotifications(1).success
-        );
-        expect(unassignSystemsModalState).toEqual({ isUnassignSystemsModalOpen: false, shouldRefresh: true, systemsIDs: [] });
-        expect(unassignSystemFromPatchSet).toHaveBeenCalledWith({ inventory_ids: ['test_1'] });
-    });
+        fireEvent.click(screen.getByText('Remove'));
```

Comment:
nit-pick: Prefer userEvent here
```suggestion
        userEvent.click(screen.getByText('Remove'));
```